src/react/databrowser/objects/details/Properties.jsx

```diff
@@ -1,5 +1,5 @@
 // @noflow
-import Table, * as T from '../../../ui-elements/TableKeyValue';
+import Table, * as T from '../../../ui-elements/TableKeyValue2';
```

Comment:
Is TableKeyValue still used after this change ? If not I think we could drop it and rename the TableKeyValue2 to TableKeyValue 🚀

Reply:
Sadly this TableKeyValue is still used by many other components :(

Reply:
Would be for next time then !